src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -26,6 +26,8 @@ def initialize()
         config_param :resend_interval_ms, :integer, :default => 30000
         desc 'timeout in millisecond when connecting to djsonsocket'
         config_param :conn_retry_timeout_ms, :integer, :default => 60000
+        desc 'the field name for the event emit time stamp'
+        config_param :emit_timestamp_name, :string, :default => "EmitTimestamp"
```

Comment:
>Emit [](start = 65, length = 4)

I think the word "Emit" might be interpreted ambiguously here... I think it might be interpreted as emitting *from* fluentd as well (note that this is an output plugin). I'd prefer another word, like 'Arrival' to 'Emit'...?